src/core/Akka/Actor/ActorPath.cs

```diff
@@ -287,6 +286,7 @@ public static bool TryParse(string path, out ActorPath actorPath)
             return true;
         }
 
+        [Obsolete]
```

Comment:
Why is `TryParseAddress` obsolete now? Or is it obsolete on `ActorPath` because we already have an equivalent on the `Address` class itself?